Hyper/PlayerData/InventorySystem/Items/Pickaxes/DiamondPickaxe.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Hyper.PlayerData.InventorySystem.Items.Pickaxes;
+
+internal class DiamondPickaxe : Pickaxe
```

Comment:
maybe we could experiment with different "building functions"? Here are some examples https://en.wikipedia.org/wiki/Bell-shaped_function 
But that's an extra

Reply:
I don't think there would be much visible difference. Maybe if we have time before we have to complete the project